libs/core/src/lib/facets/facet-group.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="fd-facet-group" role="group" [attr.aria-label]="ariaLabel">
```

Comment:
same as 'fd-facet-content'. If this is a directive the user will have the option to add labels, id, etc without fund-ngx exposing these and Input props

Reply:
Same as above,  except I suppose I can bind the class and properties directly to the host than have an extra `<div>` here so that this can also support user adding their own classes/props to the same `fd-facet-group` tag level. Is that okay?

Reply:
removed the `<div>` and this html file entirely and added the props to the host from the .ts file itself.